src/runtime/audience-action-local-flow.ts

```diff
@@ -249,6 +253,19 @@ export class AudienceActionLocalFlow implements AudienceActionFlow {
     await this.complete_();
   }
 
+  private closeOptInPrompt_() {
+    this.eventManager_.logSwgEvent(
+      AnalyticsEvent.ACTION_BYOP_NEWSLETTER_OPT_IN_CLOSE
+    );
+
+    if (this.params_.isClosable) {
+      this.unlock_();
+      if (this.params_.onCancel) {
+        this.params_.onCancel();
+      }
```

Comment:
nit: `this.param_.onCancel?.();`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls

Reply:
Done